pkg/rpc/server/server.go

```diff
@@ -539,7 +539,11 @@ func (s *Server) getApplicationLog(reqParams request.Params) (interface{}, *resp
 
 	trig := trigger.All
 	if len(reqParams) > 1 {
-		trig, err = trigger.FromString(reqParams.ValueWithType(1, request.StringT).String())
+		trigString := reqParams.ValueWithType(1, request.StringT)
+		if trigString == nil {
+			return nil, response.ErrInvalidParams
```

Comment:
What if trigger is not passed? This should work too.

Reply:
In thiss case len(reqParams) == 1, so we are handling this case 4 lines above, aren't we?